### DISCUSSION TITLE
GitHub Actions에서 멀티 플랫폼 이미지 빌드하기 | 아카이브-로그

### DISCUSSION THREAD

**Discussion by giscus** [jongwooo/blog#434] [Comment]
# GitHub Actions에서 멀티 플랫폼 이미지 빌드하기 | 아카이브-로그

멀티 플랫폼 이미지를 빌드한다는 것은 하나의 Dockerfile로 여러 플랫폼에서 실행 가능한 Docker 이미지를 생성하는 것을 의미합니다.
빌드된 이미지는 arm64, armv7, amd64, ppc64le, s390x 등의 여러 플랫폼에서 실행 가능하며, 이를 통해 다양한 플랫폼에서 동일한 이미지를 실행할 수 있습니다. GitHub Actions에서…

https://thearchivelog.dev/article/building-a-multi-platform-image-with-github-actions/

**Comment by Han-Joon-Hyeok** [comment]
역시 갓종우 👍
